index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+  <body>
+    <form action="contactForm.php" method="POST" class="contact-form">
```

Comment:
I'd consider using the built in HTML5 validation, but with custom styling. It can be a slight faff, but it makes things easier once it's set up, and you don't have to write every piece of validation yourself.